<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmLightButton</class>
 <widget class="QWidget" name="frmLightButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="LightButton" name="lightButton1"/>
   </item>
   <item>
    <widget class="LightButton" name="lightButton2"/>
   </item>
   <item>
    <widget class="LightButton" name="lightButton3"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LightButton</class>
   <extends>QWidget</extends>
   <header>lightbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
